app/Models/Ldap.php

```diff
@@ -105,6 +105,10 @@ static function findAndBindUserLdap($username, $password)
         if (!$ldapbind = @ldap_bind($connection, $userDn, $password)) {
             return false;
         }
+        
+        ldap_unbind($connection);
```

Comment:
If ldapbind fails, the function returns false, so the unbind never fires, no?

Reply:
ldap_unbind would only work if there is a successful bind, so if the bind fails, we would not want to run unbind. If it succeeds, we would want to initiate the new connection as the admin user to run the search with the respective filter query